lib/ui/painting.dart

```diff
@@ -1019,13 +1019,11 @@ enum Clip {
   antiAliasWithSaveLayer,
 }
 
-// Indicates that the image should not be resized in this dimension.
-//
-// Used by [instantiateImageCodec] as a magical value to disable resizing
-// in the given dimension.
-//
-// This needs to be kept in sync with "kDoNotResizeDimension" in codec.cc
-const int _kDoNotResizeDimension = -1;
+/// Indicates that the image should not be resized in this dimension.
+///
+/// Used by [instantiateImageCodec] as a magical value to disable resizing
+/// in the given dimension.
+const int kDoNotResizeDimension = -1;
```

Comment:
turns out we don't need to expose this, since we support "null" for this purpose as well.

Reply:
I'll go ahead and revert this part of it!

Reply:
https://github.com/flutter/engine/pull/12989